llava/eval/model_vqa_vizwiz.py

```diff
@@ -156,9 +161,11 @@ def eval_model(args):
     # Create the data loader
     data_loader = create_data_loader(questions, args.image_folder, tokenizer, image_processor, model.config)
 
+
     # Loop through the data loader and generate answers
     for (input_ids, image_tensor, image_sizes), line in tqdm(zip(data_loader, questions), total=len(questions)):
-        idx = line["question"]
+        
+        idx = line["question"] + "_" + line["image"]
```

Comment:
why is this different?

Reply:
Original LLaVA authors used a modified version of VizWiz with unique ids.

example line from llava_test.jsonl:
```
{"question_id": 0, "image": "VizWiz_test_00000000.jpg", "text": "What is this? and what color is it?\nWhen the provided information is insufficient, respond with 'Unanswerable'.\nAnswer the question using a single word or phrase.", "category": "default"}
```
However we have shifted to using the original vizwiz annotation file. Example entry looks like
```
  {
    "image": "VizWiz_test_00000000.jpg",
    "question": "What is this? And what color is it?"
  }, 
 ```
Given my work with other datasets like MME, I was worried that one image could have multiple questions asked on it, so image names are not unique enough to become an id. 

Since there is no id in the original vizwiz question, I create a makeshift id from the question content + the image directory.

The id is required to reference the same question when `scripts/convert_vizwiz_for_submission.py` is called.